src/content/blog/ai/series-outline.md

```
@@ -0,0 +1,172 @@
+description: 'Exploring the intersection of artificial intelligence and emotional wellbeing, from digital therapeutics to AI-enhanced mental health support'
+author: 'AI & Psychology Team'
+pubDate: 2024-04-26
+draft: false
+timeframe: 'May-June 2024'
+---
+
+# 🤖 AI & Emotional Intelligence Series
+
+## Series Blueprint 📋
+
+```yaml
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 120; Actual: 196

(MD013, line-length)